sound/soc/sof/amd/acp.h

```diff
@@ -201,6 +201,11 @@ extern const struct snd_sof_dsp_ops sof_renoir_ops;
 int snd_amd_acp_find_config(struct pci_dev *pci);
 
 /* Trace */
-int acp_sof_trace_init(struct snd_sof_dev *sdev, u32 *stream_tag);
-int acp_sof_trace_release(struct snd_sof_dev *sdev);
+int acp_sof_trace_register(struct snd_sof_dev *sdev);
+void acp_sof_trace_unregister(struct snd_sof_dev *sdev);
+
+/* SOF client registration */
+int acp_sof_register_clients(struct snd_sof_dev *sdev);
+void acp_sof_unregister_clients(struct snd_sof_dev *sdev);
+
```

Comment:
@ujfalusi We have tested basic trace functionality on AMD platform with this PR changes and it's working fine .. Thanks for update

Reply:
@ajitkupandey, thank you!